.github/workflows/compaund-git-tag-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Create and push git tag
+
+on:
+  workflow_call:
+    inputs:
+      runsOn:
+        required: false
+        type: string
+        default: 'ubuntu-20.04'
+      tagName:
+        required: true
+        type: string
+      tagDescription:
+        required: false
+        type: string
+        default: 'Auto genrated tag by Github Actions'
+
+jobs:
+  create-tag:
+    runs-on: ${{ inputs.runsOn }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Show tag name
+        run: |
+          echo "tag ${{ inputs.tagName }}"
+          echo "description ${{ inputs.tagDescription }}"
+      - name: Create and push tag
+        run: |
+          git config --global user.name 'Github Actions'
+          git config --global user.email 'exactpro@exactpro.com'
```

Comment:
This is usually the email that is used for auto-commits in GitHub:
name = 'github-actions'
email = '41898282+github-actions[bot]@users.noreply.github.com'

Probably, we should use this one as well